src/interfaces.ts

```diff
@@ -0,0 +1,14 @@
+import { FastifyServerOptions } from "fastify";
+
+export interface config {
+  server?: FastifyServerOptions;
+  plugins?: any[];
+  connection?: {
+    port?: string;
+  };
+  electrode?: {
+    eventTimeout?: number;
+  };
+  deferStart?: boolean;
+  keepAliveTimeout?: number;
+}
```

Comment:
please add typedoc comments to explain what the fields are use for and if there's any expected value and their effect.